packages/commonwealth/client/scripts/views/modals/edit_topic_modal.tsx

```diff
@@ -51,9 +51,18 @@ export const EditTopicModal = ({
     featuredInSidebarProp,
   );
   const [name, setName] = useState<string>(nameProp);
+  const [image, setImage] = useState<string | null>(null);
 
   const { isAddedToHomeScreen } = useAppStatus();
 
+  const markdownImageRegex = useMemo(() => /!\[image\]\((.*?)\)/, []);
+
+  //checks if there is an image in the description to show/delete separately in the modal
+  useMemo(() => {
```

Comment:
I think we don't need this useMemo either actually. If we just move the:
```typescript
const match = description.match(markdownImageRegex);
setImage(match ? match[1] : null);
```

into onDeleteImageClick, then it should update the state when you click the remove image button